.env.example

```
@@ -1,3 +1,3 @@
 NUXT_PUBLIC_API_BASE_URL=https://your-api.com
-NUXT_PUBLIC_APP_NAME=BSF App Template
-NUXT_PUBLIC_DOMAIN_ALIAS=bsfmarket
\ No newline at end of file
+NUXT_PUBLIC_APP_NAME=BSF Tutorials
```

dotenv-linter (3.3.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)